### ISSUE TITLE
Add KMS key for created resource "aws_cloudwatch_log_group" "log-management-self-log-group"

### ISSUE THREAD

**Issue by phas13** [Han-Lon/terraform-aws-cloudwatch-logs-management#1]
It is great module to manage all log groups in AWS account.

What we have noticed. log-management-self-log-group created by this module does not apply KMS key to itself, and on each apply try to remove encryption from log-management-self-log-group.

Example:

Terraform code

```
module "log_management_automation" {
  source            = "Han-Lon/cloudwatch-logs-management/aws"
  version           = "0.1.2"
  kms_key_alias     = var.kms_key_alias
  cross_regions     = "None"
  retention_in_days = "365"
}
```

Terraform plan

```
Terraform will perform the following actions:

  # module.xxx.module.cloudwatch.module.log_management_automation.aws_cloudwatch_log_group.log-management-self-log-group will be updated in-place
  ~ resource "aws_cloudwatch_log_group" "log-management-self-log-group" {
        id                = "/aws/lambda/terraform-cloudwatch-log-management-lambda"
      - kms_key_id        = "arn:aws:kms:us-east-1:123456789:key/12345678-1234-1234-1234-1234567890ab" -> null
        name              = "/aws/lambda/terraform-cloudwatch-log-management-lambda"
        tags              = {}
        # (3 unchanged attributes hidden)
    }

Plan: 0 to add, 1 to change, 0 to destroy.
```

Can you add KMS to module created log group?

**Comment by Han-Lon** [comment]
Hi @phas13, thanks for bringing this up! I missed the notification for this so it slipped under my radar, my bad. I'll check this out tomorrow in my dev environment along with [your other open issue](https://github.com/Han-Lon/terraform-aws-cloudwatch-logs-management/issues/2) and see what I can do

**Comment by Han-Lon** [comment]
Resolved in the 0.2.0 release after testing in my own environment. Please reopen this issue if it persists on your end.
